ironfish/src/rpc/routes/chain/importChain.ts

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { readBlock } from '../../../network/utils/block'
+import { IronfishNode } from '../../../node'
+import { Block } from '../../../primitives'
+import { ApiNamespace, router } from '../router'
+
+export type ImportSnapshotRequest =
+  | {
+      chunkStart?: number | null
+      chunkEnd?: number | null
+      blocks?: Buffer | null
+    }
+  | undefined
+
+export type ImportSnapshotResponse = {
+  headSeq: number
+}
+
+export const ImportSnapshotRequestSchema: yup.ObjectSchema<ImportSnapshotRequest> = yup
+  .object({
+    chunkStart: yup.number().nullable().optional(),
+    chunkEnd: yup.number().nullable().optional(),
+    blocks: yup.mixed<Buffer>().nullable().optional(),
+  })
+  .optional()
+
+export const ImportSnapshotSchema: yup.ObjectSchema<ImportSnapshotResponse> = yup
```

Comment:
Is this one used somewhere?

Reply:
Oooh, I see, the other routes do this too. I think the name should just be `ImportSnapshotResponseSchema`